lib/main.c

```diff
@@ -308,6 +335,16 @@ char *option;)
 }
 
 
+___HIDDEN ___UCS_2 lc_ctype_env_var[] =
+{'L', 'C', '_', 'C', 'T', 'Y', 'P', 'E', '\0'};
```

Comment:
I don't really know what this should be, but do you want two instances of 'C' in this list?

Reply:
yes... it is the string "LC_CTYPE"... https://www.gnu.org/software/libc/manual/html_node/Locale-Categories.html